docs/testing_policy.md

```diff
@@ -53,4 +54,51 @@ go test -v [PKG_NAME]
 Code testing occurs remotely using a [github->Actions->workflow (Build)](https://github.com/lf-edge/edge-home-orchestration-go/actions) during each `push` or `pull_request`.
 
 > [More information on github->actions](https://docs.github.com/en/actions) 
+
 ---
+
+## 4. Test file pattern
+
+Testing should include both positive and negative tests whenever possible. Tests can be nested for greater coverage and better understanding. Example:
+
+```
+package <your-package-name>
+
+import (
+	"testing"
+)
+
+const (
+	unexpectedSuccess = "unexpected success"
+	unexpectedFail    = "unexpected fail"
+)
+
+func TestFuncName(t *testing.T) {
+    // Positive test
+	t.Run("Success", func(t *testing.T) {
+		if err := FuncName(param); err != nil {
+			t.Error(unexpectedFail)
+		}
+	})
+    // Negative tests
+	t.Run("Fail", func(t *testing.T) {
+        // Nested test
+		t.Run("FailCondition #1", func(t *testing.T) {
+			if err := FuncName(param); err == nil {
+				t.Error(unexpectedSuccess)
+			}
+		})
+        // Nested test with panic result
```

Comment:
I would like to modify the indentation as below.

```suggestion
	// Positive test
	t.Run("Success", func(t *testing.T) {
		if err := FuncName(param); err != nil {
			t.Error(unexpectedFail)
		}
	})
	// Negative tests
	t.Run("Fail", func(t *testing.T) {
        	// Nested test
		t.Run("FailCondition #1", func(t *testing.T) {
			if err := FuncName(param); err == nil {
				t.Error(unexpectedSuccess)
			}
		})
		// Nested test with panic result
```

Reply:
Done! )